k2mosaic/mosaic.py

```diff
@@ -254,7 +275,13 @@ def _make_image_extension(self, extname, data):
         hdu.header['TELAPSE'] = frametim/3600./24. * num_frm
         hdu.header.cards['TELAPSE'].comment = '[d] TSTOP - TSTART'
 
-        for keyword in ['EXPOSURE', 'LIVETIME', 'DEADC', 'TIMEPIXR', 'TIERRELA',
+        hdu.header['EXPOSURE'] = int_time/3600./24. * num_frm
+        hdu.header.cards['EXPOSURE'].comment = '[d] time on source'
+
+        hdu.header['LIVETIME'] = frametim/3600./24. * num_frm * deadc
```

Comment:
This probably doesn't change anything, but it's slightly better to not repeat the computation, i.e.:
```suggestion
        hdu.header['LIVETIME'] = hdu.header['TELAPSE'] * deadc
```